PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+Fixes #Issue_Number
```

Comment:
Can we add a title here? Also this explanation about using the Fixes notation should be moved to details no? Maybe link to GitHub help for more info on this: https://help.github.com/articles/closing-issues-via-commit-messages/